config/schedule.yml

```
@@ -1,9 +1,9 @@
 create_weekly_question:
-  cron: "0 9 * * 1"
+  cron: "0 8 * * 1"
```

This will work until winter, because of daylight savings time. Instead you should tell it in which timezone you want it to run:

```suggestion
  cron: "0 9 * * 1 Europe/Lisbon"
```

---

Oh ok didn't know about this, thanks!

---

Well, according to their docs this works, but try it out.